nixos/modules/services/networking/consul.nix

```diff
@@ -8,7 +8,12 @@ let
 
   configOptions = {
     data_dir = dataDir;
-    ui = cfg.webUi;
+    ui_config = {
+      enabled = cfg.webUi;
+    };
+    telemetry = {
+      "disable_compat_1.9" = true;
```

Comment:
It might be better to stick with the upstream default here; I'd expect they will turn it off automatically in the next version.

Reply:
Agreed.